src/panels/detail/DetailLink.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import {
-    CardinalityPool,
-    Languages,
-    Links,
-    ProjectElements,
-    ProjectLinks,
-    ProjectSettings,
-    Schemes,
-    VocabularyElements
+	CardinalityPool,
```

Comment:
Is there a reason to use tabs instead of spaces for indentation? Prettier default is two spaces.